recipes/strong_type/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class StrongTypeConan(ConanFile):
+    name = "strong_type"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rollbear/strong_type"
+    description = "An additive strong typedef library for C++14/17/20"
+    topics = ("cpp14", "cpp17", "strong_type")
+    license = "Boost Software License 1.0"
```

Comment:
```suggestion
    license = "BSL-1.0"
```
https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#license-attribute